docs/source/getting-started.md

```diff
@@ -49,6 +49,7 @@ Rover is distributed on npm for integration with your JavaScript projects.
 Internally, the `npm` installer downloads router binaries from `https://rover.apollo.dev`. If this URL is unavailable, for example, in a private network, you can point the `npm` installer at another URL in one of two ways:
 
 - by setting the `APOLLO_ROVER_DOWNLOAD_HOST` environment variable
+ - *note*: This value must be set if you also want the rover plugins to be downloaded from this host
```

Comment:
```suggestion
> **Note**: If you want to download Rover plugins from the same host, you must set the `APOLLO_ROVER_DOWNLOAD_HOST` environment variable
```

Reply:
This just doesn't make sense to me, since it's a sub-bullet of the bullet that is already telling you to set the value.  I get _why_ you want to word it like that, but I think that's a bigger issue to address in a different PR now that this process will work regardless of the installation means of the rover binary.  (Right now this sub-section is just in the `npm installation` instructions.

Also, we're stuck until this merges and is included in a release, so.... pretty please!?